mesh/src/main/java/no/nordicsemi/android/mesh/sensorutils/Power.java

```diff
@@ -0,0 +1,54 @@
+package no.nordicsemi.android.mesh.sensorutils;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.RestrictTo;
+
+import java.util.Arrays;
+
+/**
+ * The Power characteristic is used to represent a measure of power in units of watts.
+ */
+public class Power extends DevicePropertyCharacteristic<Float> {
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY)
+    public Power(@NonNull final byte[] data, final int offset, final int length) {
+        super(data, offset, length);
+        int bits = 0;
+        byte[] bytes = Arrays.copyOfRange(data, offset, offset + length);
+        for (byte b : bytes) bits = (bits >> 8) | (((int) b & 0xFF) << 16);
```

Comment:
Shouldn't this be << 24?

Reply:
Also, aren't there any standard library way of doing these kind of operations?

Reply:
I think shifting with 16 is correct since it is a 24 bit number. But I have changed it to a non-loop version now. I don't know about any standard library ways of dealing with this, especially since it's a 24 bit number. But I don't know Java so ...